common/config/rush/pnpm-lock.yaml

```diff
@@ -10389,6 +10459,7 @@ packages:
       - '@swc/wasm'
       - bufferutil
       - debug
+      - encoding
```

Comment:
What's the meaning of Encoding?

Reply:
It's a package name, means the child dependency has a peer dependencies on encoding package. node-fech has a peer dependency on encoding